processor/processor.go

```diff
@@ -117,6 +117,7 @@ type HandleT struct {
 	multitenantI                   multitenant.MultiTenantI
 	backgroundWait                 func() error
 	backgroundCancel               context.CancelFunc
+	atomicAdd                      uint64
```

Comment:
I understand that this is here only for debugging purposes. If so, please make sure to cleanup afterwards